rlclientlib/vw_model/pdf_model.cc

```diff
@@ -0,0 +1,45 @@
+#include "pdf_model.h"
+#include "err_constants.h"
+#include "object_factory.h"
+#include "sampling.h"
+#include "ranking_response.h"
+#include "trace_logger.h"
+#include "str_util.h"
+
+namespace reinforcement_learning { namespace model_management {
+
+  pdf_model::pdf_model(i_trace* trace_logger) :
+    _trace_logger(trace_logger), _vw(new safe_vw("--json --quiet --cb_adf")) {
+  }
+
+  int pdf_model::update(const model_data& data, api_status* status) {
+    return error_code::success;
+  }
+
+  int pdf_model::choose_rank(uint64_t rnd_seed, const char* features, ranking_response& response, api_status* status) {
+    try {
+      // TODO: is there a simpler way to get a vw object if we do not actually need to deal with model mgmt?
```

Comment:
I am not aware of any other way.  We talked about creating a library of example parsers that lives independently of vw.  Will add an issue in vw